packages/utils/time.test.ts

```
@@ -1,13 +1,114 @@
+  describe("msleep", () => {
+    it("should sleep for the specified duration", async () => {
+      const start = Date.now();
+      await msleep(100);
+      const elapsed = Date.now() - start;
+      expect(elapsed).toBeGreaterThanOrEqual(90);
+      expect(elapsed).toBeLessThan(200);
+    });
+  });
+
+  describe("formatMsToLocal", () => {
+    beforeEach(() => {
```

ESLint

[error] Expected indentation of 2 tabs but found 4 spaces.

(`@typescript-eslint/indent`)

---

[error] Expected indentation of 2 tabs but found 4 spaces.

(indent)